docs - notes on actions (#29459)
metabase/metabase: docs/actions/introduction.md

- From a [public form](./custom.md#make-public) of an action.
- From a [button on dashboard](../dashboards/actions.md).

## Actions change data in tables, not models

> While actions are associated with models, they run their SQL code against the tables that underly the model (or even an unrelated table). Which means that anyone who has access to the underlying table(s), or to questions and models based on that data, will be able to see changes to that data made by an action. This also means that tools other than Metabase that are connected to that database will pick up the changes made by actions.

Reviewer: ```suggestion
A single action is associated with a single model. Each action will run as SQL code against the tables that underly the model (or even an unrelated table). Which means that anyone who has access to the underlying table(s), or to questions and models based on that data, will be able to see changes to that data made by an action. This also means that tools other than Metabase that are connected to that database will pick up the changes made by actions.
```

The action-model-table relationship is pretty tough to remember (I'm not sure if I understood this right yet). Might be helpful to clarify:

The action-model-table relationship is:
  - 1:1:1 for Basic actions 
  - 1:1:N for Custom actions

An action can run against an unrelated table when...? (Is this a custom action thing?)
